frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -208,11 +245,33 @@ pub mod pallet {
 		type AccountId = T::AccountId;
 		type AssetId = T::AssetId;
 		type VaultId = T::VaultId;
+		type PoolId = T::PoolId;
 
 		fn account_id() -> Self::AccountId {
 			T::PalletId::get().into_account()
 		}
 
+		#[transactional]
+		fn set_pool_id_for_asset(
+			asset_id: T::AssetId,
+			pool_id: T::PoolId,
+		) -> Result<(), DispatchError> {
```

Comment:
```suggestion
		fn set_pool_id_for_asset(asset_id: T::AssetId, pool_id: T::PoolId) -> DispatchResult {
```
In the file used `DispatchResult` everywhere. I think we should follow the same style.